script_test.go

```diff
@@ -1375,15 +1375,34 @@ func TestReadAutoCloser_ReadsAllDataFromSourceAndClosesItAutomatically(t *testin
 	}
 }
 
-func TestSliceProducesElementsOfSpecifiedSliceOnePerLine(t *testing.T) {
+func TestSlice_(t *testing.T) {
 	t.Parallel()
-	want := "1\n2\n3\n"
-	got, err := script.Slice([]string{"1", "2", "3"}).String()
-	if err != nil {
-		t.Fatal(err)
+	tests := []struct {
```

Comment:
It seems a shame to have a table test with just two cases, doesn't it? Especially as they're *different* cases, not two examples of the same behaviour.

Wouldn't you rather just structure this as two separate tests? That tends to make each test clearer, more focused, and easier to understand.

Reply:
done!